Assets/ScriptTemplates/81-C# Script__MonoBehaviour-MB.cs.txt

```diff
@@ -1,13 +1,8 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
     #ROOTNAMESPACEBEGIN#
 public class #SCRIPTNAME# : MonoBehaviour
 {
-    void Awake()
```

Comment:
i like having awake int there but i see your point
i use awake to set up dependencies (getcomponent) calls
then in start i might get properties of the cached components